lib/scriptLoader.js

```diff
@@ -17,6 +17,9 @@
 
   function webWorkerImport(loader, load) {
     return new Promise(function(resolve, reject) {
+      if (load.metadata.integrity)
+        reject('cannot verify integrity of ' + load.address);
```

Comment:
We should move this check higher up into the `lib/register.js` extension before it calls `__exec`. The check is then just if `metadata.scriptLoad` is true I believe. Also the error message may be better something like `new Error('Subresource Integrity metadata is only supported for script loads.'))` The address should display automatically with the normal error handling also I believe.